README.md

```diff
@@ -204,7 +206,7 @@ All paths below are prefixed with `solution.` for fully-qualified naming and are
 
 | Name              | Description                                                                                                                  | Default Value |
 | ----------------- | ---------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| `namespace`       | The Kubernetes namespace for CORTX Pods to be deployed in.                                                                   | `default`     |
+| `namespace`       | The Kubernetes namespace for CORTX Pods to be deployed in.                                                                   | `cortx`     |
```

Comment:
Forgive me for commenting on something that's already existing, but two nitpicks:
* We deploy third party resources (Consul, etc.) into the same namespace, not just CORTX.
* It's also not just Pods.